src/vs/workbench/services/textfile/common/textFileEditorModel.ts

```diff
@@ -480,15 +480,15 @@ export class TextFileEditorModel extends BaseTextEditorModel implements ITextFil
 		// where `value` was captured in the content change listener closure scope.
 
 		// Content Change
-		this._register(this.textEditorModel.onDidChangeContent(() => this.onModelContentChanged()));
+		this._register(this.textEditorModel!.onDidChangeContent(() => this.onModelContentChanged()));
```

Comment:
@mjbvz please avoid "`!`"

Reply:
This class is written so that methods like this are only called when `textEditorModel` is not null. Adding the null suppression was the most conservative fix. I will wrap this line in a conditional instead